packages/sources/deutsche-boerse/src/transport/proto-utils.ts

```diff
@@ -65,6 +65,31 @@ export function hasSingleOfferFrame(dat?: DataProto): boolean {
   return isDecimalPrice(dat?.Offer?.Px)
 }
 
+// true if this frame has Pxs array with MID_PRICE and PRICE_SPREAD entries
+export function hasMidPriceSpreadFrame(dat?: DataProto): boolean {
+  const pxs = dat?.Pxs
+
+  if (!pxs || !Array.isArray(pxs) || pxs.length === 0) {
+    return false
+  }
+  const hasSpread = pxs.some(
+    (entry) =>
+      entry.Typ === 8 && // MID_PRICE
+      entry.PxTyp?.Value === 12 && // PRICE_SPREAD
```

Comment:
Can we please use the Prototype enums here? 
For example: `  Data_MDEntryPrices_MDEntryType.MID_PRICE`